index.js

```diff
@@ -124,4 +126,22 @@ if (platform === 'linux') {
 	main.questionMarkPrefix = '?';
 }
 
-module.exports = platform === 'win32' ? win : main;
+var figures = platform === 'win32' ? win : main;
+
+var fn = function (str) {
+	if (figures === main) {
+		return str;
+	}
+
+	Object.keys(main).forEach(function (key) {
+		if (main[key] === figures[key]) {
+			return;
+		}
+
+		str = str.replace(new RegExp(escapeStringRegexp(main[key]), 'g'), figures[key]);
```

Comment:
Anyone has an idea why I don't need the `m` flag here? Multiline replaces seems to work even without that flag.